ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -80,7 +80,7 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
     public componentDidUpdate(): void {
         if (this.state.deep === true && this.state.workflows && this.state.workflows.length === 1) {
             const workflow = this.state.workflows[0];
-            const url = '/archived-workflows/' + workflow.metadata.namespace + '/' + (workflow.metadata.uid || '');
+            const url = uiUrl('/archived-workflows/' + workflow.metadata.namespace + '/' + (workflow.metadata.uid || ''));
```

Comment:
```suggestion
            const url = uiUrl('archived-workflows/' + workflow.metadata.namespace + '/' + (workflow.metadata.uid || ''));
```

The leading slash had to be removed here when `uiUrl` was added per https://github.com/argoproj/argo-workflows/issues/13056#issuecomment-2383564895